vdyp-sindex/src/main/Sindxdll.java

```diff
@@ -0,0 +1,3748 @@
+import java.util.NoSuchElementException;
+

```

Comment:
This should not be in the root package.  Something like
```
package ca.bc.gov.nrs.vdyp.sindex;
```
and the file path should be `vdyp-sindex/src/main/java/ca/bc/gov/nrs/vdyp/sindex/Sindexdll.java`